doc/external_resources.md

```diff
@@ -10,6 +10,7 @@ to submit a pull request with your recommended addition to the
 
 ## Tutorials
 
+- [EGU22 SC5.2: Crafting beautiful maps with PyGMT](https://github.com/GenericMappingTools/egu22pygmt) by Wei Ji Leong, Leonardo Uieda, Max Jones, Andre Belem
```

Comment:
```suggestion
- [2022 EGU SC5.2: Crafting beautiful maps with PyGMT](https://github.com/GenericMappingTools/egu22pygmt) by Wei Ji Leong, Leonardo Uieda, Max Jones, Andre Belem
```
I think this more consistent format with the items below makes it easier to read.